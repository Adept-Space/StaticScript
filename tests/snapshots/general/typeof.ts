
{
    function callTypeOfOnStaticTypes(): void {
        console_log("callTypeOfOnStaticTypes");

        console_log(typeof true);
        console_log(typeof false);
        console_log(typeof 1);
        console_log(typeof 1.0);
        console_log(typeof "hey");

        console_log("");
    }

    callTypeOfOnStaticTypes();
}
