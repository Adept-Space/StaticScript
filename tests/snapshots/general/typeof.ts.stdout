callTypeOfOnStaticTypes
boolean
boolean
number
number
string

